src/test/kotlin/com/openlattice/linking/LinkingScoringTest.java

```diff
@@ -0,0 +1,4 @@
+package com.openlattice.linking;
+
+public class LinkingScoringTest {
```

Comment:
I don't think you can use rehearsal functions here if you want to load the data first and then do the load test here Assuming this test will be for that, if it tests only the scores/is a unit test then just ignore this comment :)

Reply:
Yeah, I maybe should remove this?
It should be a unit test in the future to see if the input array of the features results in the correct score, to make sure the training order/features/... map to prod.

Reply:
You can keep it here just add a `// TODO It should be a unit test in the future to see if the input array of the features results in the correct score, to make sure the training order/features/... map to prod`